packages/atlas/src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -10,29 +10,53 @@ export default {
   title: 'other/Tooltip',
   component: Tooltip,
   argTypes: {
-    text: { defaultValue: 'Lorem ipsum dolor sit amet! Lorem ipsum dolor sit amet! Lorem ipsum dolor sit amet!' },
+    text: { defaultValue: 'Lorem ipsum dolor amet! Lorem ipsum dolor amet!' },
     headerText: { defaultValue: 'Lorem ipsum' },
-    arrowDisabled: { defaultValue: false },
     placement: {
       control: { type: 'select', options: ['top-start', 'top-end', 'bottom-start', 'bottom-end'] },
       defaultValue: 'bottom-start',
     },
     icon: { defaultValue: false },
     offsetX: { defaultValue: 0 },
     offsetY: { defaultValue: 0 },
+    className: { table: { disable: true } },
+    reference: { table: { disable: true } },
+    arrowDisabled: { table: { disable: true } },
+    footer: { control: { type: 'boolean' } },
+  },
```

Comment:
could you clean the storybook here and use `args` instead of `argTypes` to set the default values and controls?

Reply:
Also, could you add a template where the tooltip is always visible? It's really hard to inspect them with dev tools and I think it could be useful for us and for designers.